README.md

```diff
@@ -1197,6 +1197,14 @@ https://twitter.com/simalexan
 Topics: Serverless, Offline Web, Chat Bots\
 [https://twitter.com/slobodan\_](https://twitter.com/slobodan_)
 
+### Novi Sad
+
+<img src="https://avatars.githubusercontent.com/u/20664330?v=4" height="70px" width="70px" align="left" alt="" />
+
+**Stanislav Deviatov**\
+Topics: Kafka, Mulesoft, API Management, Integration Platforms \
+[https://github.com/stn1slv](https://github.com/stn1slv)
```

Comment:
we use Twitter or some other social media that allows contacting (GitHub has no messenger feature yet)

```suggestion
https://twitter.com/stn1slv
```

Reply:
@karlhorky, is it possible to specify a link to Telegram? 
https://telegram.me/stn1slv

Reply:
let's do a social media platform with public profile pages which allows for messaging the person

some Twitter alternatives I would accept are things like:

- Mastadon
- Bluesky

Reply:
Thanks for the clarifications. Twitter is ok for me.